Patterns/Chinese/Chinese-Numbers.yaml

```diff
@@ -78,34 +78,20 @@ ZeroToNineMapChs: !dictionary
     俩: 2
     倆: 2
     仨: 3
-RoundNumberMapChs: !dictionary

```

Comment:
Was this duplicated?

Reply:
Yes, it was defined on [L16](https://github.com/southworkscom/Recognizers-Text/blob/28804d1cc595248f81efd9f7e158d7ba668d91fa/Patterns/Chinese/Chinese-Numbers.yaml#L16)